cloud/deploy/deploy.go

```diff
@@ -317,8 +317,6 @@ func buildImage(c *config.Context, path, currentVersion, deployImage string, cli
 
 	if !isTagValid {
 		fmt.Println(fmt.Sprintf(warningInvalidImageTagMsg, version, isValidRuntimeVersions))
```

Comment:
Will this be throwing a warning and getting a confirmation from the user before we plan to proceed? If no, we should let them consent